src/systems/filecoin_blockchain/storage_power_consensus/storage_power_consensus_subsystem.go

```diff
@@ -65,18 +66,18 @@ func (spc *StoragePowerConsensusSubsystem_I) _getStoragePowerActorState(stateTre
 	return st
 }
 
-func (spc *StoragePowerConsensusSubsystem_I) GetTicketProductionRand(chain chain.Chain, epoch block.ChainEpoch) util.Randomness {
+func (spc *StoragePowerConsensusSubsystem_I) GetTicketProductionRand(chain chain.Chain, epoch actors.ChainEpoch) util.Randomness {
```

Comment:
nit: seeing util.Randomness in a few places, suggest deleting the type in util

Reply:
There are a ton. I decided to leave util.Randomness behind because this change was all I needed to get the actors to not depend on it, and the name `actors` seemed out of place as an import. We could follow-up to replace it with abi.Randomness after #794. 

Or I would be just as happy deleting it and replacing with `[]byte`.